src/AutoRest.CSharp/Mgmt/Models/MgmtClientOperation.cs

```diff
@@ -209,6 +198,36 @@ private FormattableString BuildDescription()
             return parameterMappings;
         }
 
+        private MethodSignature EnsureMethodSignature()
+        {
+            var attris = this._operations
+                .Where(op => Configuration.MgmtConfiguration.PrivilegedOperations.ContainsKey(op.OperationId))
+                .Select(op =>
+                {
+                    var arg = Configuration.MgmtConfiguration.PrivilegedOperations[op.OperationId];
+                    MgmtReport.Instance.TransformSection.AddTransformLog(
+                        new TransformItem(TransformTypeName.PrivilegedOperations, op.OperationId, arg),
+                        op.Operation.GetFullSerializedName(),
+                        $"Operation {op.OperationId} is marked as Privileged Operation");
+                    return new CSharpAttribute(typeof(Azure.Core.CallerShouldAuditAttribute), arg);
+                })
```

Comment:
I also have a question here:
This class is a wrapper of rest operations, there are cases that we have multiple rest operations wrapped in this same client operation.
For instance, we have two rest operations wrapped inside this client operation: `Foo_A` and `Foo_B`.
And we have the configuration assigning them both:
```
privileged-operations:
- Foo_A: "operation A"
- Foo_B: "operation B"
```

Reading the code here, this scenario, we will generate one method like this:
```csharp
[CallerShouldAudit("operation A")]
[CallerShouldAudit("operation B")]
public async Task<Response> MethodName(...)
```

Is this expected? Will this compile? By default, I believe the `AllowMultiple` on attributes is set to false, therefore a method cannot have multiple same attributes

Reply:
i thought about the scenario but decided not to handle it specially because 1. in most case, user can avoid it easily by only configuring one operation, 2. if there are cases we do want to config two for some reason, there are chance the right fix is to support AllowMultiple which is hard to determine automatically so i would prefer to let things broken and add a proper handling base on the real case then